danielMunteanu/src/main/java/danielCollectionTask/CompanyOrder.java

```diff
@@ -0,0 +1,81 @@
+package danielCollectionTask;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import static java.time.LocalDateTime.now;
+
+public abstract class CompanyOrder implements Order {
+    private final String id;
+    private Company company;
+    private List<Item> items;
+    private final LocalDateTime date;
+
+    public CompanyOrder(Company company, List<Item> items) {
+        this.id = UUID.randomUUID().toString();
+        this.company = company;
+        this.items = items;
+        this.date = now();
+    }
+
+    public String getId() {
```

Comment:
this method is never used here, can you please remove it?

Reply:
removed